tools/schematics/dt-component-dev/index.ts

```diff
@@ -23,14 +23,13 @@ import {
   template,
   Tree,
   url,
-} from '@angular-devkit/schematics';
+} from '@nrwl/workspace/node_modules/@angular-devkit/schematics';
```

Comment:
revert this change – never import from sub node_modules

Reply:
Yes, I guess I needed to do that because of the version difference in the package.json. The fix is coming up.

Reply:
I removed all sub-node_module imports.